app/src/sharedTest/java/org/oppia/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -64,6 +71,67 @@ class RevisionCardFragmentTest {
     }
   }
 
+  @Test
+  fun testReviewCardTestActivity_fractionSubtopicId1_checkReturnToTopicButtonIsDisplayedSuccessfully() {
+    launch<RevisionCardActivity>(
+      createReviewCardActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID
+      )
+    ).use {
+      onView(withId(R.id.revision_card_return_button)).check(matches(withText("Return to topic")))
```

Comment:
Add scrolling to button as first step and then click because if the content in subtopic is more that this button will be at bottom which means that this test case will fail.

Reply:
Fetch the `Return to topic` string from `strings.xml`

Reply:
Done